src/styles/embeds/sass/components/modal.css

```diff
@@ -1,21 +1,23 @@
-$modal-height: 570;
-$modal-footer-height: 90px;
+:root {
+  --modal-height: 570;
```

Comment:
There wasn't a unit in the original code, so I kept it consistent. Happy to add it in if you think its needed? Seems to work fine without